.github/workflows/translations.yml

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 Graz University of Technology.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+name: i18n:pull,extract,push translations
+on: workflow_dispatch # manually trigger 
+
+jobs:
+  translations:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # install transifex client
+      - name: transifex-client
+        run: |
+          pip install transifex-client
+
+      # store token
+      - name: store token
+        run: |
+          touch ~/.transifexrc
+          echo [https://www.transifex.com] >> ~/.transifexrc
+          echo api_hostname=https://api.transifex.com >> ~/.transifexrc
+          echo hostname=https://www.transifex.com >> ~/.transifexrc
+          echo password=${{ secrets.TRANSIFEX_TOKEN }} >> ~/.transifexrc
+          echo username=api  >> ~/.transifexrc
+
+      # pull all lang
```

Comment:
Will the changes in this pull, be part of the pull request being opened? I.e. to at least have a copy of what was on TX in case something gets overwritten?